### DISCUSSION TITLE
`uBlock filters – Cookie Notices` list multilingual?

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#18618] [Q&A] [answered]
Is your `uBlock filters – Cookie Notices` list multilingual or will it only support English sites?

**Comment by krystian3w** [comment]
Supports regional variants of pages: #18613 

---

I would only reduce the rate of adding filters like a simple `:style` filter or `set-constant.js` that will start/run normally with Easylist Cookie, mostly based on these issue moved to Fanboy Annoyances: #18559 (without address in uAssets).

**Comment by peace2000** [comment, marked as answer]
Multilingual.
